README.md

```diff
@@ -178,12 +178,12 @@ The `files` table iterates over _ALL_ the files in a commit history, by default
 The full table is every file in every tree of a commit history.
 Use the `commit_id` column to filter for files that belong to the work tree of a specific commit.
 
-| Column     | Type |
-|------------|------|
-| commit_id  | TEXT |
-| name       | TEXT |
-| contents   | TEXT |
-| executable | BOOL |
+| Column        | Type |
+|---------------|------|
+| commit_id     | TEXT |
+| file_path     | TEXT |
+| file_contents | TEXT |
```

Comment:
since this is the `files` table, let's make it `path` and `contents` for the column names. It's a different column name from stats and blame, but I think that's fine since it's implied by the name of the table, whereas in blame/stats, `file_path` makes sense since we'll want to specify what kind of path it is